Dayanand Viveki/HTML/Assignment6/assets/css/style.css

```diff
@@ -0,0 +1,43 @@
+* {
+    margin: 0px;
+    padding: 0px;
+    display: flexbox;
+}
+.font-size-35 {
```

Comment:
add underscore in class names remove - and make this change everywhere

Reply:
ok mdm